### PR TITLE
Update notification paloads for new snapshots

### PR DESCRIPTION
**Pull request by adamyonk**
## Examples

*Please note: Slack media unfurling isn't working yet, screenshots are of notifications with and without chart title, user name, snapshot message*

- [HipChat](https://cloud.githubusercontent.com/assets/33258/5728699/c7baf2aa-9b31-11e4-8f8e-76a85e01a933.png)
- [Slack](https://cloud.githubusercontent.com/assets/33258/5728698/c7b8ac0c-9b31-11e4-8d15-a9b0c7e9c450.png)
- [Campfire](https://cloud.githubusercontent.com/assets/33258/5728697/c7b8acca-9b31-11e4-96dd-b857f5bebee7.png)
- [Flowdock](https://cloud.githubusercontent.com/assets/33258/5728700/c7be4892-9b31-11e4-9e24-ab8aa2f9fb4e.png)

### CODE REVIEW

File: lib/librato-services/helpers/snapshot_helpers.rb [hunk 1]
```diff
@@ -12,7 +12,10 @@ def self.sample_snapshot_payload
             :snapshot => {
               :entity_name => "App API Requests",
               :entity_url => "https://metrics.librato.com/instruments/1234?duration=3600",
-              :image_url => "http://snapshots.librato.com/instruments/12345abcd.png"
+              :image_url => "http://snapshots.librato.com/instruments/12345abcd.png",
+              :user_email => "portal-dev@librato.com",
```
Comment (mheffner) [inline lib/librato-services/helpers/snapshot_helpers.rb]: Is there support for a common name in addition to email?

Comment (adamyonk) [inline lib/librato-services/helpers/snapshot_helpers.rb]: Mike it hasn't been mentioned. Working on a spec here: https://docs.google.com/a/librato.com/document/d/1VzeLPthNpL4rGaW19bjvKmBRC0MlUZJGIqxncfdRxmI/edit?usp=sharing

**Comment by obfuscurity** [comment]
Is there a reason for moving away from the printf style interpolation? This new layout is much harder to read imho.

**Comment by adamyonk** [comment]
Ha, I thought it was harder to read. I don't deal in this repo often, though. I can change it back if you prefer.

**Comment by obfuscurity** [comment]
I prefer the older style but @collinvandyck and @mheffner dive into the code more often than I do. I'll defer to their judgement.

**Comment by collinvandyck** [comment]
Seems reasonable. Should this be tested across all affected endpoints/services manually to ensure it's what we expect (in addition to the tests, which you've updated - thanks)?

**Comment by adamyonk** [comment]
@collinvandyck just to clarify - which seems reasonable?

And yes, I think it should be tested everywhere. I could use your help with that, my testing so far has been just pushing a test branch out to gobot with this version on it and sending a bunch of snapshots.

**Comment by collinvandyck** [comment]
For larger changes, especially ones that could affect the formattings, I personally would eyeball it in the wild (staging wild) for each service.  Pushing out test branches to gobot is the way to do it, but just checking in campfire, slack, flowdock, etc manually.

**Comment by adamyonk** [comment]
Updated PR description with screenshots of our 4 snapshot-enabled services.

**Comment by obfuscurity** [comment]
Are these images representative of the original chart (width) or are they still the squished version like we had in V1? I would :heart: to see wider snapshot images.

**Comment by adamyonk** [comment]
@obfuscurity They still use a set width. We're planning on eventually factoring in the actual chart dimensions that are set on the space that the chart is from.

**Comment by obfuscurity** [comment]
:cool:

I'm a :+1: on this but let's get @collinvandyck's approval before merging.

**Comment by collinvandyck** [comment]
+1, given that it is vetted in real life

**Pull request merged**